### PR TITLE
Build improvements + Travis CI integration

### PR DESCRIPTION
**Pull request by jensgreen**
This should solve issues #7 and #15.

Also added a a config for [travis CI](https://travis-ci.org/). It builds my fork just fine on Ubuntu and OS X: https://travis-ci.org/jensgreen/EnergyBox/builds/215189947

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: java
+
+dist: trusty
```
Comment (Rovanion) [inline .travis.yml]: Better add xenial in there too.

Comment (jensgreen) [inline .travis.yml]: Good point! Will add

Comment (jensgreen) [inline .travis.yml]: Never mind.. unfortunately it seems trusty/14.04 is the latest Ubuntu release supported by Travis.

https://github.com/travis-ci/travis-ci/issues/5821

**Comment by jensgreen** [comment]
Alright, I'll go ahead and merge this

**Pull request merged**